brainscore_language/utils/transformations.py

```diff
@@ -0,0 +1,248 @@
+import logging
+import math
+
+import numpy as np
+import xarray as xr
+from sklearn.model_selection import StratifiedShuffleSplit, ShuffleSplit, KFold, StratifiedKFold
+from tqdm import tqdm
+
+from brainio.transform import subset
+from brainscore_core.metrics import Score
+from . import fullname
+
+
+class Transformation:
```

Comment:
nomenclature comment:
I took `Transformation` to mean at the representation/data level, rather than at the evaluation level. E.g. LinReg is a transformation, and the cross validation splits are a method of evaluation. 
thoughts?

Reply:
I just thought of it as a way to re-structure data (e.g. splitting)